src/gh_pr/cli.py

```
@@ -344,5 +354,60 @@ def _handle_output(
+        from types import SimpleNamespace
+        tui_config_obj = SimpleNamespace(**tui_config)
+
+        # Launch the TUI
+        app = GhPrTUI(
+            github_service=github_client,
+            cache_service=cache_manager,
+            config=tui_config_obj
+        )
+        app.run()
+
+    except ImportError as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)